packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -325,6 +327,16 @@ class NavigationRail extends StatefulWidget {
   /// The default value is 256.
   final double? minExtendedWidth;
 
+  /// The curve for animations of the [NavigationRail].
+  /// 
+  /// The default animation is fastOutSlowIn
```

Comment:
nit: add period

Reply:
looks like the change wasn't applied.

FWIW you also want to make this a hyperlink, as in `[Curves.fastOutSlowIn].`